client/lib/string/index.js

```diff
@@ -17,6 +8,16 @@
  * @returns
  */
 export function areEqualIgnoringWhitespaceAndCase( a, b ) {
+	// Are they equal without any manipulation?
+	if ( a === b ) {
```

Comment:
null is === to null, so we should probably have the null check first?

Reply:
If a is null, and b is null, it felt right to return true - what do you think?

Reply:
Added this case to the tests.

Reply:
since we used to just explode it's not changing the contract, so... yeah i guess that's fine.